L1Trigger/L1TMuonBarrel/src/L1MuBMTEtaPatternLut.cc

```diff
@@ -101,11 +102,14 @@ cout<<"skata"<<endl;
   //                                       << file.getName() << endl; 
 
   // ignore comment lines 
-  file.ignoreLines(16);
  
+  int skip2=getIgnoredLines(file);
+  // cout<<" lines to skip2 "<<skip2<<endl;
+    file.ignoreLines(skip2);
+
```

Comment:
We do not. To read the LUT we use readString and readInteger. A comment not in the beginning could cause crash if it is on coarse and only coarse values if it is in fine.